### ISSUE TITLE
error accessing partners if no access to helpdesk

### ISSUE THREAD

**Issue by tdombos** [OCA/helpdesk#265]
On v14 If the helpdesk module is installed, but the user has no access to the helpdesk module, the access to the partner form will be denied with the following error message: 
```
You are not allowed to access 'Ticket' (helpdesk.ticket) records.

This operation is allowed for the following groups:
	- Helpdesk/Helpdesk Manager
	- Helpdesk/User
	- Helpdesk/User: Personal tickets
	- User types/Portal
```

**Comment by pedrobaeza** [comment]
You are not indicating Odoo version and you may not update to latest source code.

**Comment by dalonsod** [comment]
This issue should be solved, for v13, when #171 is merged, ant then it should be forward ported to v14.

@tdombos you can review that PR, when approved it should be merged.

**Comment by pedrobaeza** [comment]
This should be solved in latest versions
